### PR TITLE
ci(cypress): Added 3DS manual capture flow

### PR DESCRIPTION
**Pull request by likhinbopanna**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
Added three_ds manual capture flows in the cypress automation for existing connectors

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context
Three_ds manual capture flow was not present in the automation setup

## How did you test it?
Cypress

Bankofamerica 
boa

Stripe 
stripe

Nmi 
nmi
[failure cases existing issue]

Cybersource
image

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: cypress-tests/cypress/e2e/ConnectorTest/00015-ThreeDSManualCapture.cy.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                let det = getConnectorDetails(globalState.get("connectorId"))["3DS"];
+                cy.createPaymentIntentTest(createPaymentBody, det, "three_ds", "manual", globalState);
+            });
+
+            it("payment_methods-call-test", () => {
+                cy.paymentMethodsCallTest(globalState);
+            });
+
+            it("confirm-call-test", () => {
+                console.log("confirm -> " + globalState.get("connectorId"));
```
[the remark this answers is not in the document]
Comment (likhinbopanna) [reply cypress-tests/cypress/e2e/ConnectorTest/00015-ThreeDSManualCapture.cy.js]: done

**Review by Gnanasundari24 — APPROVED** [review]

File: cypress-tests/cypress/support/commands.js [hunk 2]
```diff
@@ -377,7 +379,7 @@ Cypress.Commands.add("captureCallTest", (requestBody, amount_to_capture, payment
       expect(response.body.amount_capturable).to.equal(0);
       expect(response.body.amount_received).to.equal(amount_to_capture);
       expect(response.body.status).to.equal("partially_captured");
-    }
+    }
```
Comment (pixincreate) [inline cypress-tests/cypress/support/commands.js:382]:
```suggestion
    }
```

File: cypress-tests/cypress/support/commands.js [hunk 3]
```diff
@@ -402,7 +404,6 @@ Cypress.Commands.add("voidCallTest", (requestBody, globalState) => {
     expect(response.body.status).to.equal("cancelled");
   });
 });
-
```
Comment (pixincreate) [inline cypress-tests/cypress/support/commands.js:405]: revert

File: cypress-tests/cypress/e2e/ConnectorTest/00015-ThreeDSManualCapture.cy.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+import getConnectorDetails from "../ConnectorUtils/utils";
+import State from "../../utils/State";
+import captureBody from "../../fixtures/capture-flow-body.json";
+
+let globalState;
+
+describe("Card - ThreeDS Manual payment flow test", () => {
+
+    before("seed global state", () => {
+
```
Comment (pixincreate) [inline cypress-tests/cypress/e2e/ConnectorTest/00015-ThreeDSManualCapture.cy.js:13]: remove unnecessary line

**Review by pixincreate — COMMENTED** [review]
please check formatting, other than that, LGTM.

**Review by pixincreate — APPROVED** [review]

**Review by Gnanasundari24 — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**